src/main/java/com/stripe/model/UsageRecord.java

```diff
@@ -53,4 +54,5 @@ public static UsageRecord create(Map<String, Object> params, RequestOptions opti
             subresourceURL(SubscriptionItem.class, subscriptionItem, UsageRecord.class),
             requestParams, UsageRecord.class, options);
   }
+  // </editor-fold>
```

Comment:
we should fix this one to take a subscription item as a real function param instead of the hack we have to fetch from the `params` map and deprecate this one.

Reply:
I don't like this hack either, but I think we should be consistent. Either all nested resources take their parent's ID as a separate parameter, or none of them do.

Reply:
But that's the thing, this one is not really a nested resource per se right? Since you don't go _through_ subscription item to then create the usage report.